docs/index.md

```diff
@@ -50,7 +50,36 @@ Womtool as described in the
 ```
 
 #### Sample configuration
-The sample configuration should be a YML file which adheres to the following
+
+##### CSV Format
+The sample configuration can be given as a csv file with the following 
+columns: sample, library, readgroup, R1, R1_md5, R2, R2_md5.
+
+column name | function
+---|---
+sample | sample ID
+library | library ID. These are the DNA libraries that are sequenced. Usually there is only one library per sample
```

Comment:
Since this is an RNA pipeline, people might get confused if we mention DNA here. Maybe just "libraries" is enough? Or "RNA-seq libraries"